packages/react-qr/src/ScanAddress.tsx

```diff
@@ -22,15 +22,16 @@ interface Props {
   onScan: (scanned: ScanType) => void;
   size?: string | number;
   style?: React.CSSProperties;
+  isEthereum?: boolean
 }
 
-function ScanAddress ({ className, onError, onScan, size, style }: Props): React.ReactElement<Props> {
+function ScanAddress ({ className, isEthereum, onError, onScan, size, style }: Props): React.ReactElement<Props> {
   const _onScan = useCallback(
     (data: string | null): void => {
       if (data) {
         try {
           const [prefix, content, genesisHash, ...name] = data.split(':');
-          const isValidPrefix = prefix === ADDRESS_PREFIX || prefix === SEED_PREFIX;
+          const isValidPrefix = isEthereum ? prefix === 'ethereum' : prefix === ADDRESS_PREFIX || prefix === SEED_PREFIX;
```

Comment:
```suggestion
          const isValidPrefix = isEthereum
            ? (prefix === 'ethereum')
            : (prefix === ADDRESS_PREFIX || prefix === SEED_PREFIX);
```

Just slightly cleared to read.

Reply:
I'm not sure about the other case - so the QR here does either address (where the prefix check works 100%) or the seed... not sure if the seed looks different? 

In either case, we don't seem to cater for the seed when in `ethereum` mode. If the seed is the same, it would likely be 

`(prefix === isEthereum  ? 'ethereum' : ADDRESS_PREFIX) || (prefix === SEED_PREFIX)`

Reply:
tbh I didn't know what the seed was but what you're saying makes sens

Reply:
updated